covid_graphs/covid_graphs/scatter_plot.py

```diff
@@ -5,9 +5,10 @@
 from google.protobuf import text_format
```

Comment:
scatter_plot.py should be also in scripts

Reply:
I kept it here since the it seems the graph creation logic is nontrivial. I can move it for now to scripts, but probably we should split the functionality from the command line call... unless it is not used anywhere else.

Reply:
OK, I'll refactor it next time I use it.